src/dataviews/category-dataview-model.js

```diff
@@ -29,7 +29,7 @@ module.exports = DataviewModelBase.extend({
     return params;
   },
 
-  initialize: function (attrs, opts) {
+  initialize: function (attrs, opts, state) {
```

Comment:
I think this model should not know anything about state, I'd use attributes for this, if they have a filter attribute just set it

Is there any reason to do that?